<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioChoose</class>
 <widget class="QWidget" name="AudioChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Btn_Close">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>11</y>
     <width>36</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Label_TEXT">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Start">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_quit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lable_Choose">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
